site/docs/skylark/backward-compatibility.md

```diff
@@ -53,6 +53,7 @@ guarded behind flags in the current release:
     flags](#disable-depsets-in-c-toolchain-api-in-user-flags)
 *   [Disallow using CROSSTOOL to select the cc_toolchain label](#disallow-using-crosstool-to-select-the-cc_toolchain-label)
 *   [Disallow using C++ Specific Make Variables from the configuration](#disallow-using-c-specific-make-variables-from-the-configuration)
+*   [Disallow `cfg = "data"`](#disallow-cfg--data)
```

Comment:
Have you tested that the link to the anchor works? Not sure how the formatting in the section title will affect it.

Reply:
Yes. If you click on "Files changed" then "View" you can test yourself. I also tested the committed copy in my own branch.